itou/employee_record/tests/tests_models.py

```diff
@@ -177,26 +177,42 @@ def test_archivable(self):
     def test_clone_orphan(self):
         # Check employee record clone features and properties
         good_employee_record = EmployeeRecordWithProfileFactory(status=Status.PROCESSED)
+        disabled_employee_record = EmployeeRecordWithProfileFactory(status=Status.DISABLED)
         bad_employee_record = EmployeeRecordWithProfileFactory(status=Status.PROCESSED)
-        previous_asp_id = good_employee_record.asp_id
 
+        previous_asp_id = good_employee_record.asp_id
         good_employee_record.asp_id += 1
+        good_employee_record.siret = "99999999999999"
         good_employee_record.save()
 
+        previous_disabled_asp_id = disabled_employee_record.asp_id
+        disabled_employee_record.asp_id += 1
+        disabled_employee_record.siret = "99999999999999"
+        disabled_employee_record.save()
+
         self.assertTrue(good_employee_record.is_orphan)
 
         clone = good_employee_record.clone_orphan(previous_asp_id)
         self.assertTrue(clone.pk != good_employee_record.pk)
         self.assertNotEqual(good_employee_record.created_at, clone.created_at)
         self.assertEqual(Status.READY, clone.status)
         self.assertEqual(previous_asp_id, clone.asp_id)
+        self.assertEqual(clone.job_application.to_siae.convention.siret_signature, clone.siret)
         self.assertIsNone(clone.asp_batch_file)
         self.assertIsNone(clone.asp_batch_line_number)
         self.assertIsNone(clone.asp_processing_code)
         self.assertIn(EmployeeRecord.ASP_CLONE_MESSAGE, clone.asp_processing_label)
         self.assertEqual(Status.DISABLED, good_employee_record.status)
         self.assertIsNone(clone.archived_json)
 
+        with self.assertRaises(CloningError):
+            # No such convention ID
+            disabled_employee_record.clone_orphan(disabled_employee_record.asp_id + 1)
+
+        # Cloning in DISABLED state must work
+        disabled_employee_record.clone_orphan(previous_disabled_asp_id)
+        self.assertTrue(disabled_employee_record.is_orphan)
```

Comment:
Nit: évite nombre de faux positifs. Notamment lorsque les parenthèses ont été oubliées lors de l’appel à une méthode.
```suggestion
        self.assertIs(disabled_employee_record.is_orphan, True)
```